### PR TITLE
Meaningfull project name

### PR DESCRIPTION
**Pull request by hannesa2**
What happens, when everyone has the name android ?
image

This will change this.

image

or

image

No test is needed

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -1,2 +1,3 @@
+rootProject.name = 'nextCloud'
```
Comment (AndyScherzinger) [inline settings.gradle:1]:
```suggestion
rootProject.name = 'Nextcloud'
```

...so it fits the official naming schema 👍

**Comment by AndyScherzinger** [comment]
Nice catch @hannesa2, just a minor comment about the naming schema 👍

File: settings.gradle (hunk 1 above)
Comment (hannesa2) [reply settings.gradle]: Now it is

**Review by AndyScherzinger — APPROVED** [review]

File: settings.gradle [hunk 2]
```diff
@@ -1,2 +1,3 @@
+rootProject.name = 'Nextcloud'
+
 include ':'
-//include ':nextcloud-android-library'
```
Comment (tobiasKaminsky) [inline settings.gradle:2]: Can you keep this line in, as this is "needed" if one wants to include library directly, as explained here:
https://github.com/nextcloud/android/blob/master/SETUP.md#5-how-to

**Comment by tobiasKaminsky** [comment]
Nice! 
Thanks @hannesa2 

Can you do a signed commit? (https://github.com/nextcloud/android/pull/8011/checks?check_run_id=1926214183)

**Comment by hannesa2** [comment]
> Can you do a signed commit? (https://github.com/nextcloud/android/pull/8011/checks?check_run_id=1926214183)

This makes is impossible to apply suggested changes from github web-ui

File: settings.gradle (hunk 2 above)
Comment (hannesa2) [reply settings.gradle:2]: In my point of view, this is a misleading comment, next to the `SETUP.md`, which nobody will read. 
If you really want to archive this, I suggest to make two flavors 
* `buildWithLibrary`
* `buildWithCode`

**Comment by tobiasKaminsky** [comment]
> This makes is impossible to apply suggested changes from github web-ui

Totally true. This is a missing thing on Github's side.

**Pull request merged**